stable/redis/values.yaml

```diff
@@ -13,6 +13,7 @@ serviceType: ClusterIP
 
 ## Pod Security Context
 securityContext:
+  isEnabled: true
```

Comment:
i'd suggest naming this `securityContext.enabled` as is used for other chart parameters.

Reply:
I have updated PR based on your review comment.